src/abstractdataframe/io.jl

```diff
@@ -134,11 +139,23 @@ function _show(io::IO, ::MIME"text/html", df::AbstractDataFrame;
         end
         for column_name in cnames
             if isassigned(df[!, column_name], row)
-                cell = sprint(ourshow, df[row, column_name])
+                cell_val = df[row, column_name]
+                if ismissing(cell_val)
+                    write(io, "<td><em>missing</em></td>")
+                elseif isnothing(cell_val)
+                    write(io, "<td><em>nothing</em></td>")
```

Comment:
It was nice to print `nothing` as the empty string to make the output of `decribe` more readable (and similar use cases outside of DataFrames). That's also how `nothing` is printed at the REPL.

Reply:
Ah - describe is a good point:
```
julia> df = DataFrame(a=1:2, b=["1", "2"], c=[1,nothing], d=[missing, 1])
2×4 DataFrame
│ Row │ a     │ b      │ c       │ d       │
│     │ Int64 │ String │ Union…  │ Int64?  │
├─────┼───────┼────────┼─────────┼─────────┤
│ 1   │ 1     │ 1      │ 1       │ missing │
│ 2   │ 2     │ 2      │ nothing │ 1       │

julia> describe(df)
4×8 DataFrame
│ Row │ variable │ mean    │ min     │ median  │ max     │ nunique │ nmissing │ eltype                │
│     │ Symbol   │ Union…  │ Any     │ Union…  │ Any     │ Union…  │ Union…   │ Type                  │
├─────┼──────────┼─────────┼─────────┼─────────┼─────────┼─────────┼──────────┼───────────────────────┤
│ 1   │ a        │ 1.5     │ 1       │ 1.5     │ 2       │ nothing │ nothing  │ Int64                 │
│ 2   │ b        │ nothing │ 1       │ nothing │ 2       │ 2       │ nothing  │ String                │
│ 3   │ c        │ nothing │ nothing │ nothing │ nothing │ 2       │ nothing  │ Union{Nothing, Int64} │
│ 4   │ d        │ 1.0     │ 1       │ 1.0     │ 1       │ nothing │ 1        │ Union{Missing, Int64} │
```

But note that `nothing` is printed in REPL if it is stored in a container:
```
julia> [1, nothing]
2-element Array{Union{Nothing, Int64},1}:
 1
  nothing

julia> (1, nothing)
(1, nothing)
```
and that is why I have made this change.

Given this what do you think would be best? (I would be OK to leave the old behavior and print nothing)

Reply:
I vote to continue printing `nothing` as the empty string. The inconsistency with other containers isn't great, but well...

Reply:
OK - I will change it.